libcloud/ecp.cfg.orig

```diff
@@ -56,6 +56,17 @@ libcloud:
             - zypper in -y lsb-release make gcc gcc-c++ chrony || true
             - systemctl enable chronyd.service || true
             - systemctl start chronyd.service || true
+            - sed -i 's/NETCONFIG_DNS_STATIC_SEARCHLIST=""/NETCONFIG_DNS_STATIC_SEARCHLIST="{% raw %}{{ lab_domain }}{% endraw %}"/' /etc/sysconfig/network/config
+            - sed -i 's/NETCONFIG_DNS_STATIC_SERVERS=""/NETCONFIG_DNS_STATIC_SERVERS="{% raw %}{{ nameserver_addr }}{% endraw %}"/' /etc/sysconfig/network/config
+        "sle-15.2":
+          runcmd:
```

Comment:
also you can add: `&sle15` to see-15.1 and refer it from here using `*sle15`, since you suppose to use equivalent config?